### PR TITLE
Add the Zephyr DC power meter running on the Arduino 101

### PR DESCRIPTION
**Pull request by kiendinh**
I didn't notice that I need to add your names.

I just added the DC power meter to the repo. @nagineni, @gvancuts, @poussa and @uhofemei,  please review then. Thanks.

### CODE REVIEW

**Comment by poussa** [comment]
LGTM

**Comment by gvancuts** [comment]
A couple of nitpicks above.

A more generic request, could you make the README file README.md and make it use proper markdown formatting?

File: sensors/DC_power_meter/README [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+(https://www.adafruit.com/products/904)
+
+The power values will be displayed on a Grove RGB LCD display (optional). Please
+note that the display works at 5V and requires a pull-up circuit to function. 
+https://www.seeedstudio.com/Grove-LCD-RGB-Backlight-p-1643.html
+
+The values are also transferred to the home gateway via UART line. A USB FTDI
+should be used to connect from the Arduino 101 board to the gateway.
+http://www.ftdichip.com/Products/Cables/USBTTLSerial.htm
+
```
Comment (gvancuts) [inline sensors/DC_power_meter/README]: Could we add some notes explaining the data format that we use and (more importantly) explain that it's the JS code (power-uart.js) that exposes it as an OCF server to the rest of the system?

**Comment by kiendinh** [comment]
Thanks, 
I just updated the README.md.

**Comment by gvancuts** [comment]
Thanks @kiendinh , can you update the couple of typos in main.c (s/neededm/needed) and squash all commits into one? I will merge it then. Thanks

**Comment by kiendinh** [comment]
Thanks, Geoffroy,

I just did. It says that I’m requiring to merge three commits. Without conflict, I think it easier to merge all three of them instead of squashing before merging. Could you please?

Thanks,
Kien

**Comment by kiendinh** [comment]
Thanks, @gvancuts 

I just did. It says that I’m requiring to merge three commits. Without conflict, I think it easier to merge all three of them instead of squashing before merging. Could you please?

Thanks

**Pull request merged**